src/main/java/seedu/address/model/Planus.java

```diff
@@ -72,6 +75,7 @@ public boolean hasTask(Task task) {
      */
     public void addTask(Task task) {
         tasks.add(task);
+        calendarTasks.add(task);
```

Comment:
I think this should be separated from the add task method. Because this way, deadlines will be added calendar as well I think.